### DISCUSSION TITLE
element.touchAction('tap) Deprecated /  How to replace it ?

### DISCUSSION THREAD

**Discussion by ghost** [appium/appium#20530] [Q&A]
Hello, is there a way to easily replace `element.touchAction('tap)` ?
Thanks for your help :)

**Comment by itkhanz** [comment]
Hey, you should use W3C Actions and Sequence class to perform tap actions in latest Appium.

**Comment by ghost** [comment]
Hello, thanks @itkhanz .
I found a solution, it seems that the command element.click() works fine.
On my project it works on both ANDROID and IOS
